src/WriteClient.ts

```diff
@@ -504,6 +504,7 @@ export class WriteClient<
 				// in case the document already existed during the previous step.
 				{
 					...doc.document,
+					documentTitle: doc.title,
```

Comment:
❓ #ask: Do you think it would make sense to instead update the private `updateDocument` to instead use `document.title`?

I think it could make more sense to do so as `documentTitle` only makes sense in the context of the `createDocument` method arguments 🤔

Happy to update in that direction if you're aligned!

Reply:
you mean by passing `doc` directly as a first parameter (which includes the actual document and the title)?

Reply:
Ohhh, ok, no nevermind! I thought we were spreading `doc` but we're spreading `doc.document`, all good